L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h

```diff
@@ -23,15 +26,15 @@ class HGCalConcentratorSuperTriggerCellImpl
   private:
 
     int getSuperTriggerCellId(int detid) const ;
-    static const int kSplit12_ = 0x3a;
-    static const int kSplit3_ = 0x30;
+    static std::map<int,int> kSplit_;
     static const int kWafer_offset_ = 6;
-
     static const int kSTCsize16_ = 16;
```

Comment:
This is a very minor comment, but since defining `16=16` and `4=4` does not really add information to the reader, it might be better to have something like:
```
static const int kSTCsizeCoarse_ = 16;
static const int kSTCsizeFine_ = 4;
```